app/models.py

```diff
@@ -1,3 +1,246 @@
+from django import forms
+import twitter
+import yaml
+import omdb
+import datetime
 from django.db import models
+from sentimentanalysis.analyzer import SentimentScorer
 
-# Create your models here.
+
+
+class QueryForm(forms.Form):
+    query = forms.CharField(label='Movie Title', max_length=100)
+
+
+class TwitterAPI(object):
+
+    def __init__(self):
+        # load the API keys from api_keys.txt
+        with open("scripts/twitter_api/api_keys.yml", 'r') as stream:
```

Comment:
This is going to cause an error if we deploy it. Instead of putting the keys in a file we have to set them as ```ENV``` variables in the Heroku settings. Then we have to access them as ```ENV``` variables here instead of trying to search a files for them.